### PR TITLE
IRBID in Trial Table and Adding Duration to Description

### PR DESCRIPTION
**Pull request by sylvr3**
screenshot from 2015-10-28 11 26 42
screenshot from 2015-10-28 11 26 49
screenshot from 2015-10-28 11 31 06

### CODE REVIEW

**Comment by sylvr3** [comment]
IRBID is now displayed in each row in trial table
Do we want the Duration to be its own separate field in the Add Trial modal? (see screenshot 2)
Why should the duration be added to the trial description? (Isn't it already being shown, as seen in screenshot 3?)

Thoughts @beccannlittle @ChristianMurphy @poojaRal

**Comment by ChristianMurphy** [comment]
> Do we want the Duration to be its own separate field in the Add Trial modal? (see screenshot 2)

No

> Why should the duration be added to the trial description? (Isn't it already being shown, as seen in screenshot 3?)

Duration is no longer needed in Trial description.
Instead IRB Start and IRB End dates should be show.
IRB ID should also be added to the third view.

**Comment by beccannlittle** [comment]
Duration refers to the length of time it takes a patient to complete the trial, no?

**Comment by ChristianMurphy** [comment]
That that is where I am a little unsure, last meeting Gary seemed to be wavering on whether Duration would be a constant amount of time or variable for each patient.

**Comment by sylvr3** [comment]
@beccannlittle Yeah, that was how I understood it.
@ChristianMurphy By third view, do you mean the patient surveys table?

**Comment by ChristianMurphy** [comment]
The third view being `/trail/1` The Trail view with Patients table.

**Comment by poojaRal** [comment]
Duration on the trial page shows for how long that trial is active in the
system.

File: view/partial/add-trial.hbs [hunk 1]
```diff
@@ -46,6 +46,12 @@
                     </label>
                     <input id="trial-end" class="form-control" name="endAt" type="date" required>
                 </div>
+                <div class="form-group">
+                    <label for="target-count">
+                        Target Count
+                    </label>
+                    <input type="number" min="0" name="targetCount">
```
Comment (ChristianMurphy) [inline view/partial/add-trial.hbs]: Link the label and the input `id="target-count"`
Style the input with bootstrap `class="form-control"`
and make the field `required`

**Comment by sylvr3** [comment]
screenshot from 2015-10-28 18 44 57

**Comment by sylvr3** [comment]
Resolved issue #54

**Comment by ChristianMurphy** [comment]
LGTM :shipit:

**Pull request merged**